.github/workflows/trigger-preview.yaml

```diff
@@ -10,15 +10,14 @@ on:
 jobs:
   find-pull-request:
     uses: ProjectPythia/cookbook-actions/.github/workflows/find-pull-request.yaml@main
-
   deploy-preview:
     needs: find-pull-request
     if: github.event.workflow_run.conclusion == 'success'
     uses: ProjectPythia/cookbook-actions/.github/workflows/deploy-book.yaml@main
     with:
       artifact_name: book-zip-${{ needs.find-pull-request.outputs.number }}
-      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }}  # deploy to subdirectory labeled with PR number
-      is_preview: 'true'
+      destination_dir: _preview/${{ needs.find-pull-request.outputs.number }} # deploy to subdirectory labeled with PR number
+      is_preview: "true"
       publish_dir: 'portal/_build/html'
```

Comment:
Looks like we should remove this line, as myst is leaving the built files at the default `_build/html`
```suggestion
```